### ISSUE TITLE
Must have default internet route in vpc external network for cluster using 3 nics.

### ISSUE THREAD

**Issue by leondelyon3** [F5Networks/f5-google-gdm-templates#73]
## Do you already have an issue opened with F5 support?
NO

## Description
The cluster, using 3 nics, either with lb or api is not running if default route to internet is missing in the vpc where the external subnet is configured. It seems having an internet default route in the management subnet vpc is not enough.

the bigip instances are stucked after the first boot, no reboot for switching the nics

Normally a default internet route is configured for each created vpc.

That means we need to modify the prerequisite section:

The subnets for the management **and the external networks** require a route and access to the Internet for the initial configuration to download the BIG-IP cloud library.

## Template
v3.13
Either:
https://github.com/F5Networks/f5-google-gdm-templates/tree/main/supported/failover/same-net/via-api/3nic/existing-stack/payg#prerequisites

Or:
https://github.com/F5Networks/f5-google-gdm-templates/tree/main/supported/failover/same-net/via-lb/3nic/existing-stack/payg#prerequisites

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 5

**Comment by darkrain42** [comment]
I think this affects the the 2nic deployment as well; when I was deploying those templates, "network1" needed an internet route before deploying the VMs would succeed.

**Comment by JeffGiroux** [comment]
This is most likely due to the requirement for NIC swap. During onboarding, the active interface is eth0 for mgmt. After NIC swap and a reboot, the active interface for mgmt is eth1, it does more onboarding (via internet stuff), then applies the configs to set the self IPs on remainining NICs.

As you can see, at one point the subnet associated with eth0 is in play, then the subnet associated with eth1 is at play in terms of getting stuff from the internet. I agree that the pre-reqs section should have some documentation added.

```
The subnet for the management network requires a route and access to the Internet for the initial configuration to download the BIG-IP cloud library
```

Technically that is a correct statement, but it really only applies in a 1-nic configuration. If you have multi-nic and NIC swap occuring, then you potentially need to adjust routes on multiple subnets for proper onboarding. Subnets in Google VPCs have a default route created by system. Unless user has created a custom route to some other device that is killing the outbound connection. 

https://cloud.google.com/vpc/docs/routes#routingpacketsinternet

**Comment by shyawnkarim** [comment]
Thanks for reporting this issue. We are now tracking this documentation enhancement internally with ID ESECLDTPLT-2837.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-google-gdm-templates-v2).
